code_schemes/BRA_shareable.json

```diff
@@ -0,0 +1,17 @@
+{
+  "SchemeID": "Scheme-58781eb2",
+  "Name": "BRA Shareable",
+  "Version": "0.0.0.1",
+  "Codes": [
+    {
+      "CodeID": "code-DNS-982f3ca2",
+      "CodeType": "Control",
+      "ControlCode": "DNS",
```

Comment:
Feels like Domain Name System but can't think of a better alternative Not Shareable would be great but we have a control code NS (Skip). Maybe `Has Data Privacy Risk (HDPR)`? I am okay with using DNS for this exercise but maybe we re-consider it in future requests.

Reply:
This felt like a reasonable name after the meeting we'd just had, but very happy to change to something better if we do this again once we've had more time to think about it. I'll try to do all of this as an experiment entirely in the JPLG-2020 repo so that it's easy to standardise in future. I'm not too worried about domain name system though, I don't think the RAs will find that confusing.